packages/core/src/features/gdcapi/gdcapi.ts

```diff
@@ -733,6 +733,18 @@ export const endpointSlice = coreCreateApi({
       }),
       transformResponse: (response) => response.data.hits,
     }),
+    getAnnotations: builder.query<
```

Comment:
Using the existing annotations hook was giving me an infinite loop when it got called in both the CaseView and AnnotationTable components :/